src/services/JSONDataFrame.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  Field,
+  FieldType,
+  getLinksSupplier,
+  getTimeZone,
+  LogsSortOrder,
+  PanelData,
+  sortDataFrame,
+} from '@grafana/data';
+import { getTemplateSrv } from '@grafana/runtime';
+
+import {
+  JSONDataFrameLabelsName,
+  JSONDataFrameLineName,
+  JSONDataFrameLinksName,
+  JSONDataFrameStructuredMetadataName,
+  JSONDataFrameTimeName,
+  JSONLogsScene,
+} from '../Components/ServiceScene/JSONLogsScene';
+import { getLogsPanelFrame } from '../Components/ServiceScene/ServiceScene';
+import { getJsonDerivedFieldsLinks } from './derivedFields';
+import { isLabelsField, isLabelTypesField, isLogLineField } from './fields';
+import { LabelType } from './fieldsTypes';
+import { LABELS_TO_REMOVE } from './filters';
+import { renderJSONVizTimeStamp } from './JSONViz';
+import { getLineFormatVariable } from './variableGetters';
+
+type ParsedJsonLogLineValue = string | string[] | Record<string, string> | Array<Record<string, string>>;
```

Comment:
I would just do a repo-wide string search for "Json" and get rid of them all.

Reply:
I cleaned up most (if not all) of the json/Json added in this feature, but didn't want to go full-hog and replace methods that already existed. 
F/R is not great as 'json' is in the package name, and also used in types `LogsVisualizationType`, and some local storage keys we don't want to update.